### PR TITLE
Implement ui for all blocks page

### LINKED ISSUE #729: Implement UI for all blocks page

### PR DESCRIPTION
**Pull request by prin-r**
Fix #729

### CODE REVIEW

File: scan/src/images/Images.re [hunk 1]
```diff
@@ -15,9 +15,9 @@
 [@bs.module] external report: string = "./report.png";
 [@bs.module] external copy: string = "./copy.svg";
 [@bs.module] external loadingSpinner: string = "./loading_spinner.gif";
-[@bs.module] external blockLogo: string = "./block.png";
```
Comment (taobun) [inline scan/src/images/Images.re:18]: Remove this file?

File: scan/src/pages/BlockHomePage.re [hunk 2]
```diff
@@ -72,12 +91,16 @@ let make = () => {
     <Row>
       <Col>
         <div className=Styles.vFlex>
+          <img src=Images.blockLogo className=Styles.logo />
           <Text
-            value="ALL BLOCKS"
-            weight=Text.Bold
-            size=Text.Xl
+            value="ALL SOURCES"
```
Comment (taobun) [inline scan/src/pages/BlockHomePage.re]: All BLOCKS?

File: scan/src/pages/BlockHomePage.re [hunk 3]
```diff
@@ -87,32 +110,45 @@ let make = () => {
...
+            nowrap=true
           />
         </div>
       </Col>
     </Row>
     <VSpacing size=Spacing.xl />
     <THead>
       <Row>
-        <Col> <div className=Styles.icon /> </Col>
+        <Col> <HSpacing size=Spacing.xl /> </Col>
```
Comment (taobun) [inline scan/src/pages/BlockHomePage.re]: I think padding on the left and right side of the head table feel different from design

**Review by taobun — CHANGES_REQUESTED** [review]

File: scan/src/images/Images.re (hunk 1 above)
Comment (evilpeach) [reply scan/src/images/Images.re:18]: line 27

File: scan/src/pages/BlockHomePage.re [hunk 4]
```diff
@@ -1,10 +1,12 @@
 module Styles = {
   open Css;
 
-  let vFlex = style([display(`flex), flexDirection(`row), alignItems(`center)]);
+  let vFlex = align => style([display(`flex), flexDirection(`row), alignItems(align)]);
 
   let pageContainer = style([paddingTop(`px(50))]);
```
Comment (taobun) [inline scan/src/pages/BlockHomePage.re:6]: The height of each row is too narrow when compare with design?

**Review by taobun — CHANGES_REQUESTED** [review]

File: scan/src/pages/BlockHomePage.re (hunk 4 above)
Comment (prin-r) [reply scan/src/pages/BlockHomePage.re:6]: done

**Review by taobun — DISMISSED** [review]
LGTM

**Comment by taobun** [comment]
@prin-r please fix conflict.

File: scan/src/reusable/Col.re [hunk 5]
```diff
@@ -1,7 +1,7 @@
-type alignSelf =
-  | FlexStart
+type alignment =
```
Comment (taobun) [inline scan/src/reusable/Col.re:1]: Why do we need to rename variables?

**Review by taobun — DISMISSED** [review]

**Review by taobun — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]
LSGTM!

**Pull request merged**